nautobot/core/templatetags/helpers.py

```diff
@@ -798,6 +798,19 @@ def render_job_run_link(value):
     return str(value)
 
 
+@library.filter()
+@register.filter()
+def label_list(value, suffix=""):
```

Comment:
For core team, just making sure this makes sense as a filter and with this name.